src/Process/3dTilesProcessing.js

```diff
@@ -26,6 +27,32 @@ function subdivideNode(context, layer, node, cullingTest) {
     }
 }
 
+function boundingVolumeToExtent(crs, layer, volume, transform) {
+    if (volume.region) {
+        return new Extent('EPSG:4326',
+            THREE.Math.radToDeg(volume.region[0]),
+            THREE.Math.radToDeg(volume.region[2]),
+            THREE.Math.radToDeg(volume.region[1]),
+            THREE.Math.radToDeg(volume.region[3]));
+    } else if (volume.box) {
+        const box = volume.box.clone().applyMatrix4(transform);
+        return new Extent(crs, {
+            west: box.min.x,
+            east: box.max.x,
+            south: box.min.y,
+            north: box.max.y,
+        });
+    } else {
+        const sphere = volume.sphere.clone().applyMatrix4(transform);
+        return new Extent(crs, {
+            west: sphere.center.x - sphere.radius,
+            east: sphere.center.x + sphere.radius,
+            south: sphere.center.y - sphere.radius,
+            north: sphere.center.y + sphere.radius,
+        });
+    }
+}
+
```

Comment:
Should not it be moved to Extent as well ?

Reply:
The volume part is specific to 3dtiles AFAIK. Maybe the sphere thing could be moved in a `.fromSphere`, although it'll still be possible to do it later if/when it is used elsewhere. @peppsac WDYT?